libs/fixtures/data/electionFamousNames2021/cdf-election.ts

```diff
@@ -0,0 +1,244 @@
+const cdfElection = {
+  '@type': 'BallotDefinition.BallotDefinition',
+
+  // Definition of the election (mainly candidates, contests, and ballot styles)
+  Election: [
+    {
+      '@type': 'BallotDefinition.Election',
+      ElectionScopeId: 'state-hamilton', // The GPUnit for this election
+      StartDate: '2023-02-01',
+      EndDate: '2023-02-01',
+      Type: 'general', // Other types: primary, runoff, special, etc.
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'Lincoln Municipal General Election',
+          },
+        ], // Election name in other languages would go here
+      },
+
+      // Definition of all candidates in the election across contests
+      Candidate: [
+        {
+          '@type': 'BallotDefinition.Candidate',
+          '@id': 'sherlock-holmes',
+          BallotName: {
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Sherlock Holmes',
+              },
+            ],
+          },
+          PartyId: 'party-0',
+        },
+        {
+          '@type': 'BallotDefinition.Candidate',
+          '@id': 'thomas-edison',
+          BallotName: {
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Thomas Edison',
+              },
+            ],
+          },
+          PartyId: 'party-1',
+        },
+      ],
+
+      // Definition of contests
+      Contest: [
+        {
+          '@type': 'BallotDefinition.CandidateContest',
+          '@id': 'mayor',
+          ElectionDistrict: 'district-1',
+          // TODO what is the analogous field for "section"?
+          Name: 'Mayor', // Contest name, not for ballot
+          BallotTitle: {
+            // Display name for contest on ballot
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Mayor',
+              },
+            ],
+          },
+          VotesAllowed: 1,
+          // TODO there's no field analogous to "allowWriteIns" in CDF
```

Comment:
Ideas for how to represent the "allowWriteIns" setting? What do we use it for?

Reply:
`allowWriteIns` tells us whether to put write in bubbles and spaces in a given contest when it's laid out on the ballot. We include the same number of bubbles as VotesAllowed.

With the CDF, maybe we need to lay out `CandidateOption`s with `isWriteIn` as `true`?

Reply:
Yeah I thought about that workaround. Seems reasonable